src/background.js

```diff
@@ -67,18 +67,32 @@ chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
       }
       sendResponse({err, result});
     });
```

Comment:
Added an event handler for setting the prod/dev values when the environment is set.